packages/cli/src/commands/setup/shadowenv/shadowenv.js

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs'
+import path from 'path'
+
+import boxen from 'boxen'
+import Listr from 'listr'
+
+import { getPaths, writeFile } from '../../../lib'
+import c from '../../../lib/colors'
+
+const shadowenvConfigPath = `${getPaths().base}/.shadowenv.d/rw.lisp`
+
+export const builder = (yargs) => {
+  yargs.option('force', {
+    alias: 'f',
+    default: false,
+    description: 'Overwrite existing configuration',
+    type: 'boolean',
+  })
+}
+
+export const handler = async ({ force }) => {
+  const tasks = new Listr(
+    [
+      {
+        title: 'Configuring Shadowenv...',
+        task: (_, task) => {
+          /**
+           * Check if Shadowenv config already exists.
+           * If it exists, throw an error.
+           */
+          if (!force && fs.existsSync(shadowenvConfigPath)) {
+            throw new Error(
+              'Shadowenv config already exists.\nUse --force to override existing config.'
+            )
+          } else {
+            return writeFile(
+              shadowenvConfigPath,
+              fs
+                .readFileSync(
+                  path.resolve(__dirname, 'templates', 'rw.lisp.template')
+                )
+                .toString(),
+              {
+                overwriteExisting: force,
+              },
+              task
+            )
+          }
+        },
+      },
+      {
+        title: '',
+        task: (_, task) => {
+          task.title = `One more thing...\n\n ${boxen(
+            [
+              c.green('Installation steps for Shadowenv'),
+              'See: https://shopify.github.io/shadowenv/getting-started/',
```

Comment:
I'm knew to this, too. Just saw that there's one more step after installing, which is runing `shadowenv trust`. Maybe add a line after the link above: "Then run `shadowenv trust`"

You could put a link to https://shopify.github.io/shadowenv/trust/ but they already have the site in the link above. I'll defer to your judgement.

Reply:
So I originally had this step documented in the output in c62c744c65e11763a9500834c1daddd82c90e746 but [decided](https://github.com/redwoodjs/redwood/pull/3499#discussion_r721437058) that rather than duplicate documentation from the site, I'd just link instead.